kt/godot-library/src/main/kotlin/godot/Wrapping.kt

```diff
@@ -0,0 +1,25 @@
+package godot
+
+import godot.core.KtVariant
+
+/**
+ * Used for variadic icalls.
+ * TODO: Make non generic emit and connect methods at code generation time to avoid pattern matching when calling signals
+ */
+fun wrap(obj: Any?): KtVariant {
+    if (obj == null) {
+        return KtVariant(Unit)
+    }
+    return when (obj) {

```

Comment:
I bet tristan is scratching his head when he sees this 😂

Reply:
yep but we don't have much choice regarding variadic godot methods

Reply:
Was not on all file.
Seems godot sets tabs each time I open project.